src/common/Models/Settings/PortPair.cs

```diff
@@ -47,19 +47,19 @@ public PortPair(int remotePort, string protocol = KubernetesConstants.Protocols.
         /// <summary>
         /// Local port corresponding to the remote port for tunneling
         /// </summary>
-        [JsonProperty("localPort")]
+        [JsonPropertyName("localPort")]
         public int LocalPort { get; set; }
 
         /// <summary>
         /// Remote port corresponding to the local port for tunneling
         /// </summary>
-        [JsonProperty("remotePort")]
+        [JsonPropertyName("remotePort")]
         public int RemotePort { get; set; }
 
         /// <summary>
         /// Protocol used when communicating on this port (usually TCP)
         /// </summary>
-        [JsonProperty("protocol")]
+        [JsonPropertyName("protocol")]
         public string Protocol { get; set; }
 
         /// <summary>

```

Comment:
This is new property added by recent PR, we need to update it from JsonProperty to JsonPropertyName as well. If I have time today i can try to push the change myself.

Reply:
updated the pull request (hope its okay, i just didnt want to bother you for a small change needed after merging with main)